src/assets/components.ts

```diff
@@ -0,0 +1,863 @@
+import AndroidTabBar from '@assets/components/AndroidTabBar.png'
+import IOSTabs from '@assets/components/iOSTabs.png'
+import AndroidTabs from '@assets/components/androidTabs.png'
+import AndroidSearchBar from '@assets/components/androidSearchBar.png'
+import IOSSearchBar from '@assets/components/iOSSearchBar.png'
+import IOSHomeIndicator from '@assets/components/iOSHomeIndicator.png'
+import IOSStatusBar from '@assets/components/iOSStatusBar.png'
+import AndroidStatusBar from '@assets/components/androidStatusBar.png'
+import IOSGrabber from '@assets/components/iOSGrabber.png'
+import AndroidChips from '@assets/components/androidChips.png'
+import IOSDropdownButton from '@assets/components/iOSDropdownButton.png'
+import AndroidDropdownButton from '@assets/components/androidDropdownButton.png'
+import AndroidFloatingButton from '@assets/components/androidFloatingButton.png'
+import AndroidSwitch from '@assets/components/androidSwitch.png'
+import IOSSWitch from '@assets/components/iOSSwitch.png'
+import IOSRadio from '@assets/components/iOSRadio.png'
+import AndroidRadio from '@assets/components/androidRadio.png'
+import IOSCheckbox from '@assets/components/iOSCheckbox.png'
+import AndroidCheckbox from '@assets/components/androidCheckbox.png'
+import IOSStepper from '@assets/components/iOSStepper.png'
+import IOSSlider from '@assets/components/iOSSlider.png'
+import AndroidSlider from '@assets/components/androidSlider.png'
+import IOSPageControl from '@assets/components/iOSPageControl.png'
+import IOSTextField from '@assets/components/iOSTextField.png'
+import AndroidTextField from '@assets/components/androidTextField.png'
+import IOSAccordion from '@assets/components/iOSAccordion.png'
+import AndroidAccordion from '@assets/components/androidAccordion.png'
+import IOSDrilldown from '@assets/components/iOSDrilldown.png'
+import IOSRevealDetails from '@assets/components/iOSRevealDetails.png'
+import IOSProgressBar from '@assets/components/iOSProgressBar.png'
+import AndroidProgressBar from '@assets/components/androidProgressBar.png'
+import IOSActivityIndicator from '@assets/components/iOSActivityIndicator.png'
+import AndroidActivityIndicator from '@assets/components/androidActivityIndicator.png'
+import IOSPushNotification from '@assets/components/iOSPushNotification.png'
+import AndroidPushNotification from '@assets/components/androidPushNotification.png'
+
+interface IPlatformComponent {
+  name: string
+  image?: string
+}
+
+interface IComments {
+  text: string
+  variant?: 'info' | 'warning' | 'caution' | 'plain'
+}
+interface IImpementation {
+  text: string
+  link: string
+}
+
+export interface IComponent {
+  reactNativeName: string
+  iOSComponent: IPlatformComponent
+  androidComponent: IPlatformComponent
+  reactNativeImplementation: Array<IImpementation>
+  comments: Array<IComments>
+}
+export const Components: Array<IComponent> = [
+  {
+    reactNativeName: 'dialog / modal / alert',
```

Comment:
Note that the components are divided into sections / groups: modal dialogs, lower-level views, buttons, etc. but here you squashed them all together.

We need to bring back this hierarchy (along with group descriptions).